gtsfm/utils/io.py

```diff
@@ -85,3 +84,79 @@ def save_json_file(
         json.dump(data, f, indent=4)
 
 
+def read_bal(file_path: str) -> GtsfmData:
+    """Read a BAL file.
```

Comment:
Maybe we can add parts of the docstring from write_bal or read_bal from gtsam? I added a URL there showing what BAL means

Reply:
i found it:
```
    """Write SfmData object as a "Bundle Adjustment in the Large" (BAL) file
    See https://grail.cs.washington.edu/projects/bal/ for more details on the format.
```